### PR TITLE
[TS] LPS-147086

### PR DESCRIPTION
**Pull request by jonathanmccann**
This pull is to update Spring from 5.2.20 to 5.2.22. Please let me know if I missed anything or if any of the commits look incorrect.

### CODE REVIEW

File: lib/versions-complete.xml [hunk 1]
```diff
@@ -2584,6 +2584,18 @@
 					</license>
 				</licenses>
 			</library>
+			<library>
+				<file-name>com.liferay.object.service.jar!groovy.jar</file-name>
```
Comment (dantewang) [inline lib/versions-complete.xml:2588]: @jonathanmccann People may often forget to build-lib-versions when changing third-party lib versions, and then these will sneak into the build-lib-versions of other lib version changes. Could you only commit the changes related to the Spring upgrade?

**Review by dantewang — CHANGES_REQUESTED** [review]
@jonathanmccann Please see my comment. Also, could you reorder the commits into:

1. Bump the version in the patch
2. Bump all the versions to 5.2.22
3. Auto SF
4. Build lib versions
5. FAKE GRADLE CACHE

Thanks!

**Comment by dantewang** [comment]
@catarineaguiar Could you review the CI results? Thanks!

**Comment by catarineaguiar** [comment]
I analyzed the CI results and it hasn't any regression, but needs to re-run with the new changes.

**Comment by jonathanmccann** [comment]
ci:test:acceptance

**Comment by jonathanmccann** [comment]
cit:test:sf

**Comment by dantewang** [comment]
Hi @jonathanmccann @catarineaguiar Please see https://github.com/liferay-core-infra/liferay-portal/pull/1035

**Comment by dantewang** [comment]
ci:stop

**Pull request closed without merging**